src/hooks/useContainer.ts

```diff
@@ -1,113 +1,24 @@
-import {
-  StatedBeanContainer,
-  StatedBeanApplication,
-  EffectContext,
-} from '../core';
+import { StatedBeanContainer, StatedBeanApplication } from '../core';
 import { getStatedBeanContext } from '../context';
-import { getMetadataStorage } from '../metadata';
-import { ClassType, StatedBeanMeta, StatedFieldMeta } from '../types';
+import { ClassType } from '../types';
 
 import { useState, useEffect, useContext } from 'react';
 
-function createEffectContext<Bean, Value>(
-  oldValue: Value,
-  bean: Bean,
-  beanMeta: StatedBeanMeta,
-  fieldMeta: StatedFieldMeta,
-  container: StatedBeanContainer,
-  value?: Value,
-): EffectContext {
-  return new EffectContext(
-    oldValue,
-    bean,
-    beanMeta,
-    fieldMeta,
-    container,
-    value,
-  );
-}
-
-export function useContainer<T>(
-  types: Array<ClassType<T>>,
+export function useContainer(
```

Comment:
Why generic `T` is removed?

Reply:
Because of the `types` is not an array of one same class. `unknown` is more reasonable